File: CampusServer/src/main/java/com/cai/campus/server/entity/GroupMessage.java
package com.cai.campus.server.entity;

public class GroupMessage {
    private GroupNotice notice;
    private String groupName;
    private String createUserName;

    public GroupMessage() {
    }

    public GroupMessage(GroupNotice notice, String groupName, String createUserName) {
        this.notice = notice;
        this.groupName = groupName;
        this.createUserName = createUserName;
    }

    public GroupNotice getNotice() {
        return notice;
    }

    public void setNotice(GroupNotice notice) {
        this.notice = notice;
    }

    public String getGroupName() {
        return groupName;
    }

    public void setGroupName(String groupName) {
        this.groupName = groupName;
    }

    public String getCreateUserName() {
        return createUserName;
    }

    public void setCreateUserName(String createUserName) {
        this.createUserName = createUserName;
    }
}

File: CampusClient/app/src/main/java/com/cai/campus/common/network/model/GroupMessage.kt
package com.cai.campus.common.network.model

import java.io.Serializable

class GroupMessage : Serializable  {
    var notice: GroupNotice? = null
    var groupName: String? = null
    var createUserName: String? = null

    constructor() {}
    constructor(notice: GroupNotice?, groupName: String?, createUserName: String?) {
        this.notice = notice
        this.groupName = groupName
        this.createUserName = createUserName
    }

    companion object{
        private const val serialVersionUID = 78417392165463400L
    }

}